### DISCUSSION TITLE
Not all pirctures show up (7,985 missing)

### DISCUSSION THREAD

**Discussion by EmilEmilchen** [photoprism/photoprism#4121] [Q&A] [answered]
Hi
I have a photoprism instance setup and everything seemed to work fine but the count of the pictures drastically differs between the directory with all the originals and the numbers shown in the web interface.
The webinterface shows 14,468 pictures:
![Screenshot 2024-03-25 162659](https://github.com/photoprism/photoprism/assets/70268797/e6d82623-4b4c-4c1b-92a6-170b039cbdda)
![Screenshot 2024-03-25 162709](https://github.com/photoprism/photoprism/assets/70268797/e5885b1c-ce95-4ee4-bb3b-d67bd5732f3e)
while there should be 22,453 pictures:
![Screenshot 2024-03-25 163138](https://github.com/photoprism/photoprism/assets/70268797/eda1d3c9-eab2-4f38-b6e2-5c6568e784fd)
I would greatly appreciate any help as I have no idea why this is happening and I'm really annoyed! Thanks in advance!
Regards, Emil

**Comment by lastzero** [comment]
Library > Originals shows actual files, whereas Search counts unique photos and videos:
- https://docs.photoprism.app/user-guide/faq/

Also see:
- https://docs.photoprism.app/getting-started/troubleshooting/#missing-pictures
- https://github.com/photoprism/photoprism/issues/3518
- https://github.com/photoprism/photoprism/issues/4118

**Reply by EmilEmilchen** [reply]
Thank you for the quick reply, that makes sense. But there are still 19850 unique JPGs:
![Screenshot 2024-03-25 171548](https://github.com/photoprism/photoprism/assets/70268797/f08ff0b2-a4d1-42d8-a0f7-386da15c9052)
(i hope i dindnt miss anything)

**Reply by lastzero** [reply]
They could be stacked, depending on your settings. Also make sure to check the filesystem permissions. Did you check the logs and review section yet?

**Reply by EmilEmilchen** [reply]
I coppied them all at once from an external drive so to me it wouldnt make any sense to me if the permissions differed. There werent any errors after a Complete rescan (in the webinterface) and running `docker compose logs -f --tail=100`  and starting to index (not a complete rescan for now as that would take about 8 hours, I didn't think it would make a difference) results in: 
```
photoprism-1  | time="2024-03-25T16:20:38Z" level=info msg="indexing originals..."
photoprism-1  | time="2024-03-25T16:20:38Z" level=info msg="classify: loading nasnet"
photoprism-1  | time="2024-03-25T16:20:42Z" level=info msg="classify: loading labels from labels.txt"
photoprism-1  | time="2024-03-25T16:20:43Z" level=info msg="index: found no .ppignore file"
photoprism-1  | time="2024-03-25T16:20:43Z" level=info msg="index: found no new or modified files"
photoprism-1  | time="2024-03-25T16:20:43Z" level=info msg="index: updated 0 files [4.937872957s]"
photoprism-1  | time="2024-03-25T16:21:14Z" level=info msg="indexing completed in 35 s"
```

Edit: in the review section there are 95 pictures, but these are already included in the number from my seconds screenshot in my initial post

Edit 2: I'm not sure what you mean by "stacked"

**Reply by lastzero** [reply]
Are you sure all of these are unique and none are duplicates? See:
- https://docs.photoprism.app/user-guide/library/duplicates/

**Reply by EmilEmilchen** [reply]
The number of originals is also too low, as far as I understand at least the number of originals should be equivalent to the number of all the pictures and ` find /media/photos/photos-origin/ -type f -iname '*.jpg' -exec sha1sum {} + | awk '{print $1}' | sort | uniq | wc -l` (I think this should reuturn the number of JPGs with unique SHA1 checksums (as said here: [https://docs.photoprism.app/user-guide/library/duplicates/](url))) returns 19,850:
![Screenshot 2024-03-25 173916](https://github.com/photoprism/photoprism/assets/70268797/a4c3e7b0-b5c6-48b9-8fa2-abe199c3bfe9)

**Reply by lastzero** [reply]
Not if the files are duplicates, please see:
- https://docs.photoprism.app/user-guide/library/duplicates/

**Reply by lastzero** [reply]
It can also be a mix of this, so duplicates, invalid files, and so on... did you go through the complete checklist we provide?

**Reply by lastzero** [reply, marked as answer]
Files in hidden folders would also be skipped, for example. It doesn't look like your commands exclude these? See:
- https://docs.photoprism.app/user-guide/library/originals/#ignoring-files-and-folders

**Reply by lastzero** [reply]
As mentioned in our documentation, it can also help to reduce the log level to see additional information.

**Reply by EmilEmilchen** [reply]
I'm sorry, I didn't think of hidden files, when checking again with ` -not -path '*/\.*'` option the number fits perfectly! Thank you so much for the help, greatly appreciated!
